rad/prelude/set.rad

```diff
@@ -17,14 +21,31 @@
     (delete x s)))
 
 (def member?
-  "Query if an value is an element of a set."
-  (fn [x s]
-    (member? x s)))
+  "Query if a value is an element of a set."
```

Comment:
Looks like the fn def was lost?

Reply:
Sets are implemented as dicts so `set/member?` is just an alias for `member?`.

Reply:
Hm, somehow didn't see the next line.